File: java/aula03/exemplo2-desvio-composto/src/exemplo2/desvio/composto/Exemplo2DesvioComposto.java
package exemplo2.desvio.composto;

import java.util.Scanner;

public class Exemplo2DesvioComposto {

    public static void main(String[] args) {
        int idade;
        Scanner entrada = new Scanner (System.in);
        
        System.out.println("Informe a sua idade: ");
        idade = entrada.nextInt();
        
        if (idade >= 18) {
            System.out.println("Maior de idade!");
        } else {
            System.out.println("Menor de idade!");
        }
    }
    
}

File: Python/aula3/exemplo2-desvio-composto.py
# Solicita a idade do usuário
idade = int(input("Digite sua idade: "))

# Desvio composto exibe mensagem “Maior de idade” quando a idade for maior ou igual a 18 anos
# e “Menor de idade” caso contrário
if idade >= 18:
    print("Maior de idade")
else:
    print("Menor de idade")